website/docs/src/luigi-config/extended/algoliaSearcher.js

```diff
@@ -0,0 +1,71 @@
+import algoliasearch from "algoliasearch";
+
+class AlgoliaSearcher{
+
+  constructor() {
+    this.client = algoliasearch("BH4D9OD16A", "5ab04e0673d89f07c964afcf1522ad3a");
```

Comment:
Just a question. Why do we have the AppId here and not in the former version? Is it neccessary?

Reply:
In former version, AppId was in the package dependency : 

file:website/docs/node_modules/docsearch.js/dist/cdn/docsearch.js
Line 2687        appId = _ref$appId === undefined ? 'BH4D9OD16A' : _ref$appId,

Reply:
Done